toolkit/tools/internal/logger/writerhook.go

```diff
@@ -59,10 +59,8 @@ func (h *writerHook) Fire(entry *logrus.Entry) (err error) {
 	}
 
 	if !h.useColors {
-		entry.Message = strings.ReplaceAll(entry.Message, "\x1b[31m", "")
-		entry.Message = strings.ReplaceAll(entry.Message, "\x1b[34m", "")
-		entry.Message = strings.ReplaceAll(entry.Message, "\x1b[0m", "")
-		entry.Message = strings.ReplaceAll(entry.Message, "\x1b[32m", "")
+		re := regexp.MustCompile(`\x1b\[[0-9]+m`)
```

Comment:
Please move this to a global private variable (see: examples elsewhere where we use `MustCompile`). There's no need to re-compile the regular expression every time we enter this function. Which may be often, since it's fired for every log we take. At least that's my understanding.